README.md

```diff
@@ -1,35 +1,21 @@
-# ATM-3
-
-All The Mods 1.12
-======
-This is the official repository and issue-tracker for All The Mods 3 1.12
-  
-Does "All The Mods" *really* contain ALL THE MODS? No, of course not.
-  
-#### Curse Links: 
-+ [![All the Mods 1](http://cf.way2muchnoise.eu/242462.svg "ATM1") All The Mods](https://mods.curse.com/modpacks/minecraft/242462-all-the-mods)  
-+ [![All the Mods 3](http://cf.way2muchnoise.eu/269708.svg "ATM3") All The Mods 3](https://mods.curse.com/modpacks/minecraft/269708-all-the-mods-3)  
-+ [![All the Mods 3 Lite](http://cf.way2muchnoise.eu/274129.svg "ATM3: Lite") All The Mods 3 Lite](https://minecraft.curseforge.com/projects/atm-3-lite)  
-+ [![All the Mods Expert Remastered](http://cf.way2muchnoise.eu/274849.svg "ATMER") All The Mods Expert Remastered](https://mods.curse.com/modpacks/minecraft/252034-all-the-mods-expert)  
-  
-Need Help?
-======
-When reporting an issue put the version number before the issue title! Such as [FULL][1.37] My game is broken! Also include any added mods you may have put in, into the description of the issue. 
- 
-You can also find us on Discord for help or just to chat as well as Reddit:   
-  
-|              |
-|:------------:|
-|<a href="https://discordapp.com/invite/rbSZNDQ"><img src="https://discordapp.com/assets/fc0b01fe10a0b8c602fb0106d8189d9b.png" alt="Join us on Discord!"  width="200" height="68"></a>|
-|<a href="https://www.reddit.com/r/allthemods"><img src="https://www.redditstatic.com/about/assets/reddit-logo.png" alt="/r/AllTheMods on Reddit"  width="200" height="67"></a>|
-<br>
-<br>
-
-### Money / Donation
-
-All The Mods has never been about making money, we just love playing games and making cool modpacks and contributing to the modded community. We understand that sometimes people like to "say thanks" so by request we have setup a Patreon with some perks and are considering expanding some ATM branded merchandise as well. Any money that we do get is used to "give back" such as hosting free public ATM servers   
-
-<a href="https://www.patreon.com/allthemods">
-	<img src="https://c5.patreon.com/external/logo/become_a_patron_button.png" alt="Support us on Patreon"  width="200" height="47">
-</a>  
+[![Logo Image](https://i.imgur.com/R3RgVCP.png)](https://minecraft.curseforge.com/members/ATMTeam/projects)
```

Comment:
Shouldn't the ATM3 logo be used rather than the AllTheMods title?

Reply:
I figured ATM is a team, and should have it's logo representing the pack.

Reply:
![image](https://user-images.githubusercontent.com/16361449/45422737-696c0480-b691-11e8-965a-331331727219.png)
Uncentered, messy colors

Reply:
whoops, meant to reply this to the comment on the badges

Reply:
Honestly centering the image would be a easier if it was a scalable vector, do you guys have an actual location where I can get all the assets for ATM? I removed the contributors badge and changed the color for the discord badge, it's less "messy" colors now and match the header image, also is now centered with the current size of the header image. 

If you can provide more constructive feedback on the changes I make in the future, this process would be much easier for me to understand what you would like changed. Vague critic without a providing a recommendation to your idea of a change makes it difficult to understand you idea of what you want.

Reply:
To center it, can't you simply use <center> ... </center>? If not, you could inline CSS it to make it centered.

I'll check the badges later.

Reply:
Never was a fan of using CSS and/or HTML in a base markdown file, doesn’t look clean and looks extremely out of place.